server/webapp/WEB-INF/rails.new/app/assets/javascripts/package_material_check_connection.js

```diff
@@ -15,18 +15,22 @@
  *************************GO-LICENSE-END**********************************/
 
 PackageMaterialCheckConnection = function (url) {
-
     _bind = function (parent_container_selector, check_connection_selector, connection_message_selector) {
         jQuery(parent_container_selector).on('click', check_connection_selector, function (e) {
             var connection_message_element = jQuery(connection_message_selector);
-            connection_message_element.removeClass("error_message").removeClass("ok_message").text('Checking connection...');
+            var formData = jQuery(parent_container_selector + " :input").serialize();
+            formData = formData.replace("_method=PUT", "_method=POST");
+
             jQuery.ajax({
-                type: "GET",
+                type: "POST",
```

Comment:
You can write a spec to test it 
https://github.com/gocd/gocd/blob/4a8fbd842355b4a1b82f85211c95001e7b215c46/server/webapp/WEB-INF/rails.new/spec/controllers/api/server_state_controller_spec.rb#L35